lib/withInNativeApp.js

```diff
@@ -3,9 +3,17 @@ import { HttpHeaderContext } from './apollo/withData'
 
 export const matchUserAgent = value => value && value.match(/RepublikApp/)
 
-export const runInApp = process.browser && matchUserAgent(navigator.userAgent)
-  ? callback => callback()
-  : () => {}
+export const runInApp = (appCallback, browserCallback) => {
```

Comment:
There is a subtle difference between `withInNativeApp` and `runInApp`. `inNativeApp` is true while server rendering. `runInApp` does not run while server rendering for the app (important in the `withMe` case).

We should clean up the naming, make it more consitent and explicit. What do you think about adding `browser` or `webview` to `runInApp`?
```js
export const runningInNativeAppBrowser = process.browser && matchUserAgent(navigator.userAgent)
// or just inNativeAppBrowser? naming is soo hard
export const runInNativeAppBrowser = runningInNativeAppBrowser
  ? () => callback()
  : () => {}
```
_also notice the small difference of only evaluating `process.browser` and `navigator.userAgent` once, it does not change after the first evaluation_

(and then use the new `runningInNativeAppBrowser` export instead of a branch like function)

Reply:
I like it. Between `runningInNativeAppBrowser` and `inNativeAppBrowser` I like the most the latter.
However, IMO `AppBrowser` is a bit redundant. Maybe the most descriptive name would be `runninginNativeApp`. I can change it, but all options are good for me 😄